### DISCUSSION TITLE
Idea: To reduce build time

### DISCUSSION THREAD

**Discussion by ryadav96** [shuding/nextra#1141] [Ideas]
I would like to suggest adding a feature to disable the generation of static pages for a specific directory during the build process. Instead, we could generate these pages on demand and cache them for future requests. 
This feature could be configured using the meta.json file. 

If this feature has already been implemented, please let me know how we can go about implementing it in our current setup. I am currently conducting a proof of concept using this template.

**Comment by shuding** [comment]
I think this is related to versioning too where we want to use lazy SSG: https://github.com/shuding/nextra/issues/20#issuecomment-1246170431

**Reply by ryadav96** [reply]
Thank you for suggesting that I review issue #20. I have done so and it does seem related to my suggestion.
